apps/storybook/src/comboboxes/Listbox.stories.tsx

```diff
@@ -184,6 +184,21 @@ export const IconMultiSelect: Story = {
   },
 };
 
+export const ItemWithIcon: Story = {
```

Comment:
let's rename this to clarify that this is just rendering icons inside the content/children
```suggestion
export const ComplexContent: Story = {
```